roles/hyperfoil_controller/tasks/retrieve_stats.yml

```diff
@@ -15,7 +15,7 @@
   when: download_results
 - name: Download JFR
   ansible.builtin.fetch:
-    src: "{{ hyperfoil_dir }}/hfc-{{ ansible_hostname }}.jfr"
+    src: "{{ hyperfoil_dir | default('/tmp/hyperfoil') }}/hfc-ip-{{ ansible_default_ipv4.address | replace('.', '-') }}.jfr"
```

Comment:
Think there might have been a disconnect with what I meant about having the default be /tmp/hyperfoil. I meant to set the `hyperfoil_dir` variable in https://github.com/infinispan/infinispan-ansible-benchmark/blob/main/group_vars/all.yaml to be /tmp/hyperfoil this way it is picked up by all of the roles. Then we can remove all usages of /tmp/hyperfoil and replace it with {{ hyperfoil_dir }}

Reply:
Updated.